src/com/session/GoogleResp.java

```diff
@@ -17,45 +20,58 @@
 	private static final long serialVersionUID = 1L;
 
 	public void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException {
-		
-		String accessToken=null;
+
 		String authcode = req.getParameter("code");
-		String email=null;
-		boolean isPresent=false;
-		System.out.println(authcode);
+		String error = req.getParameter("error");
+		GoogleInfo post = null;
+
+		if ("access_denied".equals(error)) {
+			resp.sendRedirect("login?access_denied");
```

Comment:
indeed two possible values, but my question is if Error is not null and also its not access_denied, lets say for eg: "server_error", are we handling it ?